### DISCUSSION TITLE
QTableView

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#488] [Ideas]
![image](https://github.com/user-attachments/assets/0809d9d2-1073-4fbe-b894-71de393dd1d4)
![image](https://github.com/user-attachments/assets/93f9fc84-bdbd-490e-9ab3-eceeb27b6aeb)

If I want to implement QTableView what should I do, inherit QskListView?

**Comment by uwerat** [comment]
qquicktableview.cpp ( the implementation of the QML table view ) has ~7.5K lines of code - something that can't be reimplemented or wrapped easily. That's why it is not on the agenda for the first release of QSkinny.

A list view is something different: see QListView vs. QTableView. If the list view is good enough for your needs - yes you have to inherit from QskListView and bind your data to the cells.
